packages/vkui/src/components/FormItem/FormItem.e2e-playground.tsx

```diff
@@ -58,7 +63,7 @@ export const FormItemPlayground = (props: ComponentPlaygroundProps) => {
         },
```

Comment:
Давай вынесем отдельно, чтобы не генерировать такое кол-во вариантов

```suggestion
        },
        {
          top: [
            'Сверху будет находиться достаточно длинное название текста с очень большим количеством однотипных слов для переполнения текста',
          ],
          topMultiline: [undefined, true],
        },
```

Reply:
Вынес